pageserver/src/tenant.rs

```diff
@@ -1042,21 +1051,12 @@ impl Tenant {
     /// Subroutine of `load_tenant`, to load an individual timeline
     ///
     /// NB: The parent is assumed to be already loaded!
-    #[instrument(skip(self, local_metadata, ctx), fields(timeline_id=%timeline_id))]
     async fn load_local_timeline(
         &self,
         timeline_id: TimelineId,
```

Comment:
Not sure why Dmitry removed this `#[instrument]`.

Reply:
Walking up the call graph, there is a `tenant_id` set, but not `timeline_id`.
Fixing that.